hail_scripts/v02/utils/computed_fields/variant_id.py

```diff
@@ -20,14 +20,20 @@ def get_expr_for_contig_number(table):
     )
 
 
-def get_expr_for_original_alt_alleles_set(split_variant):
-    """Compute variant id for each original alt allele in a variant split with hl.split_multi"""
-    return hl.set(
-        split_variant.old_alleles[1:].map(
-            lambda alt: get_expr_for_variant_id(hl.min_rep(split_variant.old_locus, [split_variant.old_alleles[0], alt]))
-        )
-    )
```

Comment:
I believe this breaks gnomAD's pipeline for loading ExAC.

Reply:
I had been thinking about changing these functions to accept individual field expressions rather than the full table, since passing the table requires a specific table structure. I like `get_expr_for_variant_ids` more than `get_expr_for_orig_alt_alleles` since it's more general. If `get_expr_for_variant_ids` took a locus and alleles argument, then the ExAC pipeline could pass old_locus and old_alleles.

Reply:
I like that idea.